misc/fibonacci_triangle.c

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file fibonacci_triangle.c
```

Comment:
Better to remove the filename, otherwise we'll need to add the full directory.
```suggestion
 * @file
```